bolt/include/bolt/Core/DIEBuilder.h

```diff
@@ -124,9 +125,10 @@ class DIEBuilder {
   std::vector<std::unique_ptr<DIEAbbrev>> Abbreviations;
   BinaryContext &BC;
   DWARFContext *DwarfContext{nullptr};
-  bool IsDWO{false};
+  DWARFUnit *SkeletonCU{nullptr};
```

Comment:
Please split out `IsDWO` -> `isDWO` into an NFC change for it to be reviewed/accepted faster.

Reply:
I don't think it simplifies the review, and context for change is within this PR anyway.

Reply:
It greatly simplifies the review for reviewers like me who don't have the context of the problem area and have to make an effort to tell related and unrelated things apart. And second, it's within LLVM developer policy guidelines https://llvm.org/docs/DeveloperPolicy.html#incremental-development:

> Each change should be kept as small as possible. This simplifies your work (into a logical progression), simplifies code review and reduces the chance that you will get negative feedback on the change. Small increments also facilitate the maintenance of a high quality code base.